docs/content/guides/columns/column-filter.md

```diff
@@ -129,8 +123,8 @@ const hot = new Handsontable(container, {
   rowHeaders: true,
   filters: true,
   dropdownMenu: ['filter_by_condition', 'filter_action_bar'],
-  height: 'auto',
-  licenseKey: 'non-commercial-and-evaluation'
+  height: "auto",
+  licenseKey: "non-commercial-and-evaluation"
```

Comment:
I thought we were switching to double quotes only for JSX, and sticking with single quotes for JS, right?

Reply:
True, my "find and replace" went a little too far here.